packages/dds/tree/src/shared-tree/treeCheckout.ts

```diff
@@ -317,6 +318,7 @@ export function createTreeCheckout(
 		args?.removedRoots,
 		args?.logger,
 		args?.breaker,
+		args?.disposeForksAfterTransaction,
```

Comment:
Don't forget to forward this value down whenever a checkout is forked. Otherwise, it will only be set for the root/main checkout.

Reply:
passed down flag for when forking